pkg/proxy/config/config.go

```diff
@@ -290,8 +287,7 @@ func NewNodeConfig(ctx context.Context, nodeInformer v1informers.NodeInformer, r
 
 	handlerRegistration, _ := nodeInformer.Informer().AddEventHandlerWithResyncPeriod(
 		cache.ResourceEventHandlerFuncs{
-			AddFunc:    result.handleAddNode,
```

Comment:
you are removing the Add event handler, so is not really merging add an update

Reply:
hm... it seems we don't have unit tests of this

Reply:
(confirmed that the version of this without the AddFunc fails the new unit test)